File: ManagemateAngular/src/app/shared/interfaces/API_Input_Models/Company_Models/Add_Company_Data.ts
import {Session_Data} from "../../API_Other_Models/Session_Models/SessionData";

export interface Add_Company_Data{
    session: Session_Data;

    name: string;
    surname: string;
    company_name: string;
    nip: string;
    phone_number: string;
    email: string;
    address: string;
    city_id_FK: number;
    postal_code: string;
    bank_name: string;
    bank_number: string;
    web_page: string;
    money_sign: string
}

File: Managemate/ManagemateAPI/ManagemateAPI/Management/M_Company/Input_Objects/Add_Company_Data.cs
using ManagemateAPI.Management.M_Session.Input_Objects;

namespace ManagemateAPI.Management.M_Company.Input_Objects
{
    public class Add_Company_Data
    {
        public Session_Data session { get; set; }

        public string name { get; set; }

        public string surname { get; set; }

        public string company_name { get; set; }

        public string NIP { get; set; }

        public string phone_number { get; set; }

        public string email { get; set; }

        public string address { get; set; }

        public long city_id_FK { get; set; }

        public string postal_code { get; set; }

        public string bank_name { get; set; }

        public string bank_number { get; set; }

        public string web_page { get; set; }

        public string money_sign { get; set; }

        public string money_sign_decimal { get; set; }

        public byte[]? company_logo { get; set; }

        public string? file_type { get; set; }

    }
}
